Sources/ArcGISToolkit/Components/OverviewMap.swift

```diff
@@ -185,3 +185,15 @@ private extension Symbol {
         )
     )
 }
+
+/// A very basic data model class containing a Map.
+public class MapDataModel: ObservableObject {
+    /// The `Map` used for display in a `MapView`.
+    @Published public var map: Map
+    
+    /// Creates a `MapDataModel`.
+    /// - Parameter map: The `Map` used for display.
+    public init(map: Map) {
+        self.map = map
+    }
+}
```

Comment:
I think we'll need the equivalent of this for `Scene`.
<img width="842" alt="Screenshot 2022-12-16 at 12 10 16" src="https://user-images.githubusercontent.com/16397058/208181136-508ace33-879c-4ba9-934d-1c4775d65312.png">

Also, should this be marked `public`? Otherwise I think the ones added in Examples/Utility are redundant. Not sure why a duplicate symbol error isn't produced.

Reply:
I made the Example models internal and the OverviewMap model private. The `Cannot find 'SceneDataModel' in scope` error shouldn't happen, as the `SceneDataModel` is now in the Examples app.

Reply:
Hmm, still having issues here.
<img width="430" alt="Screenshot 2022-12-16 at 13 17 05" src="https://user-images.githubusercontent.com/16397058/208190497-570bb6ce-4c53-40de-959e-7725a27f99b7.png">

I think the app level `MapDataModel` and `SceneDataModel` aren't being compiled, because they're not in Compile Sources and now that `MapDataModel` in `OverviewMap` is private, non of the example structs can resolve it.

<img width="783" alt="Screenshot 2022-12-16 at 13 21 06" src="https://user-images.githubusercontent.com/16397058/208191049-5ec768c0-81b6-4d6e-8351-cd406b69f476.png">

Reply:
Ref #224